### DISCUSSION TITLE
启动《Minecraft For Windows 10》 和MC地下城

### DISCUSSION THREAD

**Discussion by wuzhuoyi114514** [Meloong-Git/PCL#1412] [提问] [answered]
启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城启动《Minecraft For Windows 10》 和MC地下城

**Comment by WForst-Breeze** [comment, marked as answer]
刷屏哒咩~！😶
PCL2 不会支持基岩版和其他 MC 衍生版本的启动服务，请见 #81

**Comment by Chiloven945** [comment]
这是java版启动器……
